webrender/Cargo.toml

```diff
@@ -22,6 +22,7 @@ fnv = "1.0"
 gleam = "0.4.7"
 lazy_static = "0.2"
 log = "0.3"
+nanovg = {version = "0.3", features = ["gl2"]}
```

Comment:
This will need to be an optional dependency because we won't be using it in Gecko